packages/babel-traverse/test/fixtures/regression/11576/output.js

```diff
@@ -0,0 +1,5 @@
+foo
+/* WORKING */
+(
+  /* NOT WORKING */
```

Comment:
We should only consider comments in params as `innerComments` of `CallExpression` when it does not have param, otherwise the comment should be categorized as leading / trailing comments of adjacent parameters.

CI [fails](https://github.com/babel/babel/pull/12560/checks?check_run_id=1606912639#step:6:671) because`/* comment 1*/ in input source
```
fn(a, b, /* comment 1 */) /* comment 2*/;
```
was now an innerComment. Generally Babel avoids generating `innerComments` because where they should be inserted can be vague.